db/migrate/20191002224831_add_category_to_institutions.rb

```diff
@@ -0,0 +1,7 @@
+class AddCategoryToInstitutions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :institutions, :category, :string
+    Institution.update_all(category: "university")
```

Comment:
it looks like there's some states and cities currently in the database (or at least in my kind of old local copy), so some individual edits might be necessary